server/src/common.constants/getPosts.constants.ts

```diff
@@ -0,0 +1,2 @@
+export const postsPerPage = 9;
```

Comment:
Pls write literal constant in format
```
const POST_PER_PAGE = 9;

```
and next one

Reply:
Fixed.